magento/magento2-page-builder
# Broken CMS block preview when using identifier in pagebuilder

### Description (*)
When adding a CMS block in a CMS page using the block identifier instead of the block ID, pagebuilder displays following error : "Block with ID: [block identifier] doesn't exist." instead of showing preview of the CMS block.

The block renders properly on front-end.
There is no way to add a block with his identifier directly from pagebuilder. This issue would only happen if a developer executes a data patch that generates CMS content or directly edit the content in the database.

### Bug
* [<2838>](https://jira.corp.magento.com/browse/<2838>) Broken CMS block preview when using identifier in pagebuilder

### Fixed Issues (if relevant)
1. magento/magento2#35070: Broken CMS block preview when using identifier in pagebuilder

### Manual testing scenarios (*)

1. Create a CMS Block with any content
2. Create a CMS Page and add previously created CMS Block in the page with the PageBuilder Block component
3. Open cms_page table
4. Find the created page
5. Replace block_id value with the block identifier in the content cell. Example for a block with identifier contact-us-banner :
6. <div data-content-type="block" data-appearance="default" data-element="main">{{widget type="Magento\Cms\Block\Widget\Block" template="widget/static_block/default.phtml" block_id="contact-us-banner" type_name="CMS Static Block"}}</div>
7. Apply changes
8. Refresh pagebuilder page to see error

### Checklist
